pkgs/tools/networking/modemmanager/default.nix

```diff
@@ -93,6 +94,21 @@ stdenv.mkDerivation rec {
   '';
   installCheckTarget = "check";
 
+  passthru = {
+    # provided FCC unlock scripts. Used by the NixOS module system to symlink
+    # to them from /etc/ModemManager/fcc-unlock.d/….
+    # Most of them actually symlink to a "common" unlock script
+    fccUnlockScripts = {
+      "03f0:4e1d" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/1199";
+      "105b:e0ab" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/105b";
+      "1199:9079" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/1199";
+      "1eac:1001" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/1eac";
+      "2c7c:030a" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/2c7c";
+      "413c:81a3" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/1199";
+      "413c:81a8" = "${modemmanager}/share/ModemManager/fcc-unlock.available.d/1199";
+    };
```

Comment:
Please add a check to `passthru.tests` ensuring the code is kept in sync (see e. g. NetworkManager or fwupd packages).

Reply:
I don't think modemmanager will include more unlock scripts ever. It looks like that vendors will provide their own custom unlock scripts, rather than playing this whack-a-mole game, as least when reading https://modemmanager.org/docs/modemmanager/fcc-unlock/, which also lists these as an (exhaustive) list.

I'd personally prefer dropping all of `enableBundledFccUnlockScripts`, and leave it up to the user to explicitly set their unlock script (in case more SAR-aware scripts pop up somewhere) over trying to make this update-proof. I only kept this for backwards-compat with the previous NixOS option.

Reply:
I agree with @flokli on this one, @jtojnar do you think this is a blocking matter?

Reply:
Just to make sure, my /preference/ would be to entirely drop the auto enabling of the bundled FCC unlock scripts, and document how to enable the bundled ones in the release notes. I'm happy to update the PR to that, but we need to decide which direction to go.

Reply:
Sorry, I thought I have already replied that I am fine with removing the `enableBundledFccUnlockScripts` but apparently not.

Reply:
No problem, thanks for confirming. I'll update the PR.